client/jetpack-connect/signup.js

```diff
@@ -164,6 +165,10 @@ export class JetpackSignup extends Component {
 			} );
 			return;
 		}
+		if ( error && error.error && 'password_invalid' === error.error ) {
```

Comment:
What about `if ( get(  error, [ 'error' ] ) === 'password_invalid' ) {`?

Reply:
Nice 👍 Updated in https://github.com/Automattic/wp-calypso/pull/27285/commits/4e897f5d9fe95b1be581a18570f9ffa7a47f7ee9.